### PR TITLE
Pimp readme to cover more cases

### PR DESCRIPTION
**Pull request by jensens**
Some additions to the generated README.

### CODE REVIEW

File: volto_addon/{{ cookiecutter.addon_name }}/README.md [hunk 1]
```diff
@@ -11,7 +11,11 @@
 
 <!-- List your awesome features here -->
 
-## Install
+## Installation
+
```
Comment (stevepiercy) [inline volto_addon/{{ cookiecutter.addon_name }}/README.md:15]:
```suggestion

To install your project, you must choose the method appropriate to your version of Volto.

```

File: volto_addon/{{ cookiecutter.addon_name }}/README.md [hunk 2]
```diff
@@ -11,7 +11,11 @@
...
+### Into a @plone/generator-volto Volto based project
+
+This is the official way to do it in Volto 17 and before.
```
Comment (stevepiercy) [inline volto_addon/{{ cookiecutter.addon_name }}/README.md]:
```suggestion
```

File: volto_addon/{{ cookiecutter.addon_name }}/README.md [hunk 3]
```diff
@@ -11,7 +11,11 @@
 
 <!-- List your awesome features here -->
 
-## Install
+## Installation
+
+### Into a @plone/generator-volto Volto based project
```
Comment (stevepiercy) [inline volto_addon/{{ cookiecutter.addon_name }}/README.md]: I don't think the implementation detail is helpful for most users and will just confuse most of them. Let's stick to a Volto version, unless there is a very good reason to use an implementation detail.

```suggestion
### Volto 17 and earlier
```

File: volto_addon/{{ cookiecutter.addon_name }}/README.md [hunk 4]
```diff
@@ -45,19 +49,47 @@ Start volto with:
 yarn start
 ```
 
+### Into a cookiecutter-volto based project
```
Comment (stevepiercy) [inline volto_addon/{{ cookiecutter.addon_name }}/README.md]:
```suggestion
### Volto 18 and later
```

File: volto_addon/{{ cookiecutter.addon_name }}/README.md [hunk 5]
```diff
@@ -45,19 +49,47 @@ Start volto with:
...
+
+This is the way to install it in the (upcoming) Volto 18.
```
Comment (stevepiercy) [inline volto_addon/{{ cookiecutter.addon_name }}/README.md]:
```suggestion
```

File: volto_addon/{{ cookiecutter.addon_name }}/README.md [hunk 6]
```diff
@@ -45,19 +49,47 @@ Start volto with:
...
+
+Add `{{ cookiecutter.npm_package_name }}` to your `package.json`:
+
+```JSON
```
Comment (stevepiercy) [inline volto_addon/{{ cookiecutter.addon_name }}/README.md]:
````suggestion
```json
````

File: volto_addon/{{ cookiecutter.addon_name }}/README.md [hunk 7]
```diff
@@ -45,19 +49,47 @@ Start volto with:
...
+}
+```
+
+Add `{{ cookiecutter.npm_package_name }}` to your `volto.config.js`:
+
+```javascript
+const addons = ['{{ cookiecutter.npm_package_name }}'];
+```
+
+If this package provides a volto theme, and you want to activate it (and only then!), add to your `volto.config.js`:
```
Comment (stevepiercy) [inline volto_addon/{{ cookiecutter.addon_name }}/README.md]:
```suggestion
If this package provides a Volto theme, and you want to activate it, then add the following to your `volto.config.js`:
```

File: volto_addon/{{ cookiecutter.addon_name }}/README.md [hunk 8]
```diff
@@ -45,19 +49,47 @@ Start volto with:
...
+
+```javascript
+const theme = '{{ cookiecutter.npm_package_name }}';
+```
+
 ### Test it
```
Comment (stevepiercy) [inline volto_addon/{{ cookiecutter.addon_name }}/README.md]:
```suggestion
## Test installation
```

File: volto_addon/{{ cookiecutter.addon_name }}/README.md [hunk 9]
```diff
@@ -45,19 +49,47 @@ Start volto with:
...
 
 Go to http://localhost:3000/, login and check the awesome new features.
```
Comment (stevepiercy) [inline volto_addon/{{ cookiecutter.addon_name }}/README.md]:
```suggestion
Visit http://localhost:3000/ in a browser, login, and check the awesome new features.
```

File: volto_addon/{{ cookiecutter.addon_name }}/README.md [hunk 10]
```diff
@@ -45,19 +49,47 @@ Start volto with:
...
 
+
 ## Development
 
 The development of this add-on is done in isolation using a new approach using pnpm workspaces and latest `mrs-developer` and other Volto core improvements.
 For this reason, it only works with pnpm and Volto 18 (currently in alpha)
 
-### Requisites
+### Preprequisites
```
Comment (stevepiercy) [inline volto_addon/{{ cookiecutter.addon_name }}/README.md]: Wait, are these pre-requisites required only for development or for creating a project as well? If the latter, then this section must be moved immediately before Installation.

```suggestion
### Pre-requisites
```

File: volto_addon/{{ cookiecutter.addon_name }}/README.md [hunk 11]
```diff
@@ -45,19 +49,47 @@ Start volto with:
...
+
+- **Node** Node has to be installed, e.g. using the [Node Version Manager (nvm)](https://github.com/nvm-sh/nvm).
+  Ensure to have the latest LTS version installed and used.
+- **Make** [GNU Make](https://www.gnu.org/software/make/) is required to run the Makefile. You can install it using your package manager.
+- **Docker** [Docker](https://docs.docker.com/get-docker/) is required to easily run the backend server.
```
Comment (stevepiercy) [inline volto_addon/{{ cookiecutter.addon_name }}/README.md]: Replace all the above with links to our docs.

-   [Node.js](https://6.docs.plone.org/install/create-project.html#node-js)
-   [Make](https://6.docs.plone.org/install/create-project.html#make)
-   [Docker](https://6.docs.plone.org/install/create-project.html#docker)

**Review by stevepiercy — CHANGES_REQUESTED** [review]

File: volto_addon/{{ cookiecutter.addon_name }}/README.md (hunk 10 above)
Comment (jensens) [reply volto_addon/{{ cookiecutter.addon_name }}/README.md]: This are the per-requisites for development.

**Review by stevepiercy — APPROVED** [review]
I made a couple of commits to clean up the README.md parts that were outside this PR, as well as replaced the pre-requisite links with links to Plone 6 documentation.

File: volto_addon/{{ cookiecutter.addon_name }}/README.md [hunk 12]
```diff
@@ -45,19 +50,45 @@ Start volto with:
...
+"dependencies": {
+    "{{ cookiecutter.npm_package_name }}": "*"
+}
+```
+
+Add `{{ cookiecutter.npm_package_name }}` to your `volto.config.js`:
+
+```javascript
+const addons = ['{{ cookiecutter.npm_package_name }}'];
+```
```
Comment (davisagli) [inline volto_addon/{{ cookiecutter.addon_name }}/README.md:67]: Isn't there a missing step to install the new package after adding it to package.json?

Comment (jensens) [reply volto_addon/{{ cookiecutter.addon_name }}/README.md:67]: `make install` probably.

**Review by ericof — APPROVED** [review]

**Pull request merged**